modules/server/src/mapping/mappingToFields.js

```diff
@@ -1,11 +1,12 @@
 import { capitalize } from 'lodash';
 
-import mappingToNestedFields from './mappingToNestedFields';
-import mappingToScalarFields from './mappingToScalarFields';
 import createConnectionTypeDefs from './createConnectionTypeDefs';
+import mappingToNestedFields from './mappingToNestedFields';
 import mappingToObjectTypes from './mappingToObjectTypes';
+import mappingToScalarFields from './mappingToScalarFields';
 
-const mappingToFields = ({ type, parent }) => {
+const mappingToFields = ({ enableAggregationMode, type, parent }) => {
+	const fieldsToExclude = enableAggregationMode ? ['hits'] : [];
```

Comment:
followup note: in a quick conversation over zoom, Ciarán clarified for me that this feature is meant to cover the case where it may be desirable to limit the functionality of a node to only provide aggregations. (i.e. a limited data node)